cmake/setup_write_config.cmake

```diff
@@ -56,6 +57,10 @@ MACRO(_summary)
   # Only write to summary.log:
   FILE(APPEND ${_log_summary} "${ARGN}")
 ENDMACRO()
+MACRO(_featurelog)
```

Comment:
Ah -- yes, please do it everywhere while you're there (or in a follow-up if you want). It's hard to read otherwise.